src/libConsensus/ConsensusLeader.h

```diff
@@ -57,7 +57,7 @@ class ConsensusLeader : public ConsensusCommon {
   unsigned int m_numOfSubsets;
   // Received commits
   std::mutex m_mutex;
-  std::atomic<unsigned int> m_commitCounter;
+  std::atomic<unsigned int> m_commitCounter{0};
```

Comment:
Any idea why `clang-tidy` initializes this as `{0}`  but for other cases it's just `{}`?

Reply:
manual error, will fix